docs/resources/new_in_doc.md

```diff
@@ -1,12 +1,47 @@
 ---
 description: Overview of major recent additions to Ibexa DXP documentation.
-month_change: true
```

Comment:
I'd leave it as true - this way the label in TOC shows that there is new stuff in the doc.